scripts/validate_urls.sh

```diff
@@ -24,5 +24,6 @@ softwarearchitekt.at,\
 https://plus.google.com/+AngularJS,\
 https://blog.craftlab.hu/multiple-solutions-for-angular-ahead-of-time-aot-compilation-c474d9a0d508,\
 https://www.bitsrc.io
```

Comment:
Missing a comma?

Reply:
I think I made a mistake. Sorry
But a link wasn't available, then I added it to whiteList

Reply:
Yes, but you're missing `,\` at the end of the line.